pkg/controllers/routercerts/controller_test.go

```diff
@@ -342,6 +391,20 @@ func withServer(host string, options ...func([]*cryptoMaterials) []*cryptoMateri
 	}
 }
 
+func withServerCN(host string, options ...func([]*cryptoMaterials) []*cryptoMaterials) func([]*cryptoMaterials) []*cryptoMaterials {
```

Comment:
`withServerNoSAN` would be a better name

Reply:
ok; PTAL